namelayer-spigot/src/main/resources/plugin.yml

```diff
@@ -1,6 +1,6 @@
 name: NameLayer
 main: vg.civcraft.mc.namelayer.NameLayerPlugin
-version: ${project.version}
+version: ${releaseVersion}
```

Comment:
For the moment "sort of" -- I'll review further and work it out. Right now you would never change the spigot or bungee poms, but you'd change the parent pom and the "all" pom.